generators/java/Test/JavaGeneratorSerializeTest.java

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Since this is a new source, just remove the NEM headers. No need to use ProximaX headers.

Reply:
but that was merge from nemtech branch. I brought in the java generator as created by nemtech. if you want we can keep it in separate branch but that adds some overhead of the long-living branch